charts/ccd-data-store-api/values.yaml

```diff
@@ -72,7 +72,7 @@ java:
     OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
     CCD_DRAFT_STORE_URL: http://draft-store-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CCD_DEFAULTPRINTURL: https://return-case-doc-ccd.nonprod.platform.hmcts.net/jurisdictions/:jid/case-types/:ctid/cases/:cid
-    CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
i guess these are temporary changes only

Reply:
Yes - I have had to raise a temp PR for case-document-am,, that points back to this PR.  These changes will be reverted before the merge

Reply:
why do we need that 🤔 ?

Reply:
👍

Reply:
Fixed in 458e277

Removed final check of metadata in ccd-case-document-am-api